### PR TITLE
Fix duplicate in history

### PR DESCRIPTION
**Pull request by antoine-de**
There seems to be duplicate in the history (especially in the [tisseo dataset](https://transport.data.gouv.fr/datasets/tisseo-offre-de-transport-gtfs/), but I manually cleaned them.

With this we should never add a duplicate in the history

### CODE REVIEW

File: apps/transport/lib/transport/history.ex [hunk 1]
```diff
@@ -44,20 +44,21 @@ defmodule Transport.History do
...
-      |> Enum.max_by(fn {date, _hash} -> date end, fn -> nil end)
-
-    case max_last_modified do
-      nil ->
-        true
-
-      {max_last_modified, content_hash} ->
-        modification_date = modification_date(resource)
-        max_last_modified < modification_date && content_hash != resource.content_hash
+    if length(backuped_resources) == 0 do
```
Comment (Tristramg) [inline apps/transport/lib/transport/history.ex]: `Enum.empty?`

File: apps/transport/lib/transport/history.ex [hunk 2]
```diff
@@ -44,20 +44,21 @@ defmodule Transport.History do
...
+      true
+    else
+      already_there = backuped_resources
+      |> Enum.find(fn r -> r.content_hash == resource.content_hash end)
```
Comment (Tristramg) [inline apps/transport/lib/transport/history.ex]: I _think_ if there is only one element, put it in the function call instead of `|>`

Comment (Tristramg) [reply apps/transport/lib/transport/history.ex]: And I’d use again an Enum.empty? instead of comparing against nil

Comment (antoine-de) [reply apps/transport/lib/transport/history.ex]: hum I don't get how you want to use Enum.empty? here.
the Enum.find function return nil is not found

Comment (Tristramg) [reply apps/transport/lib/transport/history.ex]: Yup, sorry

**Pull request merged**